src/Components/Button.js

```diff
@@ -0,0 +1,12 @@
+import React, { Component } from 'react';
+
+export default class Button extends React.Component {
+  render() {
+    const { name } = this.props;
+    return (
+      <div className="Button">
+        <button type="button">{name}</button>
+      </div>
+    );
+  }
+}
```

Comment:
Add prop validation for each props in a component
Button.propTypes = {name: PropTypes.string.isRequired,}